components/org.wso2.carbon.identity.user.endpoint/src/main/resources/api.identity.user.yaml

```diff
@@ -124,6 +124,84 @@ paths:
         401:
           description: Unauthorized request
 
+  /me/resend-code:
+    post:
+      description: |
+        This API is used to resend the confirmation code to the authenticated user if it is missing.
+      x-wso2-request: |
+        curl -X POST -H "Authorization: Basic <Base64Encoded_username:password>" -H "Content-Type: application/json" -d '{"properties": [{"key":"RecoveryScenario","value": "SCENARIO_NAME"}]}' "https://localhost:9443/api/identity/user/v1.0/me/resend-code"
+      x-wso2-curl: |
+        curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"properties": [{"key":"RecoveryScenario","value": "MOBILE_VERIFICATION_ON_UPDATE"}]}' "https://localhost:9443/api/identity/user/v1.0/me/resend-code"
+      x-wso2-response: |
+        "HTTP/1.1 201 Created"
+      summary: |
+        Resend code
+
+      # This are the post parameters:
```

Comment:
```suggestion
      # These are the post parameters:
```

Reply:
Addressed in https://github.com/wso2-extensions/identity-governance/pull/414/commits/b5f0c45e2ce7cb40a24f7a051acc52455755cce4